CONTRIBUTING.md

```diff
@@ -19,6 +19,26 @@ Alternatively, you can use your own local WordPress environment and clone this r
 
 Next, open a terminal (or if on Windows, a command prompt) and navigate to the repository you cloned. Now type `npm install` to get the dependencies all set up. Then you can type `npm run dev` in your terminal or command prompt to keep the plugin building in the background as you work on it.
 
+### Hot reloading
+
+Webpack hot module reloading (hmr) with is available for the editor. While developing this allows you to see your changes without having to refresh the page.
+
+To enable webpack hot module reloading, place the following line in `wp-config.php` (in the root of your WordPress installation):
+
+```php
+define( 'GUTENBERG_WEBPACK_HMR', true );
+```
+
+Build the project ( `npm run build` ), and start the hmr development server by running `npm run hot`. Any changes to files used by the editor ( `/editor` ), should be reflected in the browser without reloading the page.
```

Comment:
- Should it be a requirement to `npm run build` first, or can this be part of the `hot` script?
- Should HMR be built into `npm run dev` ?
- At first I was confused into thinking `/editor` was the URL path, not the relative path from the repository. Not sure I have any suggestions here, aside from either assigning as a link to the absolute repository path or prefixing to clarify relative filesystem path (`./editor`)